includes/managers/widgets.php

```diff
@@ -234,16 +234,21 @@ public function register_widget_type( Widget_Base $widget ) {
 	 *
 	 * @param \Elementor\Widget_Base $widget_instance Elementor Widget.
 	 *
-	 * @return true True if the widget was registered.
+	 * @return bool True if the widget was registered.
 	 * @since 3.5.0
 	 * @access public
-	 *
 	 */
 	public function register( Widget_Base $widget_instance ) {
 		if ( is_null( $this->_widget_types ) ) {
 			$this->init_widgets();
 		}
 
+		$should_register = apply_filters( 'elementor/widgets_manager/is_widget_registered', true, $widget_instance );
```

Comment:
```suggestion
		$should_register = true;
		
		/**
		 * Should widget be registered.
		 *
		 * @since 3.18.0
		 *
		 * @param bool $should_register Should widget be registered. Default is `true`.
		 * @param \Elementor\Widget_Base $widget_instance Widget instance.
		 */
		$should_register = apply_filters( 'elementor/widgets/is_widget_enabled', $should_register, $widget_instance );
```

Why? because we have the `elementor/widgets/register` hook.